<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>720</width>
    <height>489</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GetAllFromVIew</string>
  </property>
  <property name="statusTip">
   <string/>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QPushButton" name="activateFunc_btn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>작업시작</string>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Query_LineEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>471</width>
     <height>20</height>
    </rect>
   </property>
   <property name="contextMenuPolicy">
    <enum>Qt::DefaultContextMenu</enum>
   </property>
   <property name="toolTip">
    <string>Input Search Keyword</string>
   </property>
   <property name="statusTip">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Keyword</string>
   </property>
  </widget>
  <widget class="QPushButton" name="AddQuery_btn">
   <property name="geometry">
    <rect>
     <x>503</x>
     <y>98</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextBrowser" name="FileDIR_Browser">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>471</width>
     <height>45</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="CheckQuery_btn">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>Keywords확인 및 변경</string>
   </property>
   <property name="text">
    <string>CheckQuery</string>
   </property>
  </widget>
  <widget class="QPushButton" name="SaveAsFIle_btn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>400</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Save As File</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="ViewResult_table">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>471</width>
     <height>192</height>
    </rect>
   </property>
  </widget>
  <widget class="QCheckBox" name="FlagEditable_chbox">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>250</y>
     <width>85</width>
     <height>16</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>더블클릭시 수정가능</string>
   </property>
   <property name="text">
    <string>결과 수정</string>
   </property>
  </widget>
  <widget class="QComboBox" name="SelectKeyword_cbox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>471</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="changeAdmit_btn">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>280</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>적용</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="openFile_btn">
   <property name="geometry">
    <rect>
     <x>503</x>
     <y>32</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
